.github/workflows/ci.yml

```
@@ -86,5 +86,16 @@ jobs:
+          cache: "pip"
+          cache-dependency-path: nexum_ai/requirements.txt
+
+      - name: Install dependencies
+        run: pip install -r nexum_ai/requirements.txt
+
+      - name: Byte-compile nexum_ai
+        run: python -m compileall nexum_ai
+
       - name: Byte-compile nexum_ai (syntax check)
         run: python -m compileall -q nexum_ai
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)